transports/janus_http.c

```diff
@@ -35,6 +35,13 @@
 #include <netdb.h>
 
 #include <microhttpd.h>
+#ifdef HAVE_ENUM_MHD_RESULT
+	/* enum MHD_Result introduced in libmicrohttpd v0.9.71 */
+	typedef enum MHD_Result MHD_Result;
+#else
+	typedef int MHD_Result;
+#endif
```

Comment:
Shouldn't this just be an `#ifndef HAVE_ENUM_MHD_RESULT` to define it to `int` for older MHD versions? Won't this redefine something that exists already, for newer versions, and possibly cause issues?

Reply:
I guess MHD just uses `enum MHD_Result`, but doesn't typdef's it as `MHD_Result`, correct?

Reply:
Verified looking at `microhttpd.h` that they don't indeed typedef `MHD_Result`, so no issue in your code there.